submissions/adeolaFafemi/phpDatabases/connection.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+try{
+    $pdo = new PDO("mysql:host=localhost;dbname=under9K_store", "root", "nickiminaj_20");
+    
+    $pdo -> setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+    $pdo -> getAttribute(constant("PDO::ATTR_CONNECTION_STATUS"));
+    return $pdo;
+    }catch(PDOException $e){
```

Comment:
Format this block of code @kojusola

Reply:
Done 
Thank you

Reply:
From line 3 to 10 isn't formated properly